build-support/bin/generate_github_workflows.py

```diff
@@ -198,6 +198,38 @@ def checkout(
     return steps
 
 
+def launch_bazel_remote() -> Sequence[Step]:
+    return [
+        {
+            "name": "Launch bazel-remote",
+            "if": "github.repository_owner == 'pantsbuild'",
+            "run": dedent(
+                """\
+                mkdir -p ~/bazel-remote
+                docker run -u 1001:1000 \
+                  -v ~/bazel-remote:/data \
+                  -p 9092:9092 \
+                  buchgr/bazel-remote-cache \
+                  --s3.auth_method=access_key \
+                  --s3.secret_access_key="${AWS_SECRET_ACCESS_KEY}" \
+                  --s3.access_key_id="${AWS_ACCESS_KEY_ID}" \
+                  --s3.bucket=cache.pantsbuild.org \
+                  --s3.endpoint=s3.us-east-1.amazonaws.com \
+                  --max_size 30 \
+                  &
```

Comment:
I think `docker run` supports a `--detach` argument... I'm assuming it's the same as just putting it in the background like this, but here's a comment just in case docker does something special/better with it?

```suggestion
                  --detach
```

Reply:
Good idea (athough the `--detach` must come after `docker run`, not as part of the process args)